ui/src/main/js/MainPage.js

```diff
@@ -12,6 +12,8 @@ import { getDescriptors } from 'store/actions/descriptors';
 import DescriptorContentLoader from 'dynamic/loaded/DescriptorContentLoader';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import ProviderTable from 'providers/ProviderTable';
+import SlackGlobalConfiguration from 'channels/SlackGlobalConfiguration';
+import { SLACK_INFO } from 'channels/SlackModels';
```

Comment:
It might also be nice to make the channel files more organized with a /slack directory.